modin/pandas/series_utils.py

```diff
@@ -292,6 +292,13 @@ def findall(self, pat, flags=0, **kwargs):
             query_compiler=self._query_compiler.str_findall(pat, flags=flags, **kwargs)
         )
 
+    def fullmatch(self, pat, case=True, flags=0, na=np.NaN):
```

Comment:
```suggestion
    def fullmatch(self, pat, case=True, flags=0, na=None):
```

Why `np.NaN` when pandas 1.4.4 uses `None `as default? https://github.com/pandas-dev/pandas/blob/ca60aab7340d9989d9428e11a51467658190bb6b/pandas/core/strings/accessor.py#L1257-L1288